scripts/dev.sh

```diff
@@ -179,6 +179,7 @@ export REQUIRE_USER_AUTHENTICATION=$REQUIRE_USER_AUTHENTICATION
 export REQUIRE_USER_PERMISSIONS=$REQUIRE_USER_PERMISSIONS
 export DEBUG=$DEBUG
 export MAGE_DATA_DIR=$MAGE_DATA_DIR
+export SERVER_LOGGING_TEMPLATE=$SERVER_LOGGING_TEMPLATE
```

Comment:
we can remove this, i don't think it's necessary

Reply:
Done